src/enhancedScatterChartSettingsModel.ts

```diff
@@ -19,27 +21,27 @@ export enum DisplayUnitsType {
     Thousands = 1000,
     Millions = 1000000,
     Billions = 1000000000,
-    Triilions = 1000000000000
+    Trillions = 1000000000000
 }
 
 const displayUnitsOptions : IEnumMember[] = [
-    {value: DisplayUnitsType.Auto, displayName : DisplayUnitsType[DisplayUnitsType.Auto]},
-    {value: DisplayUnitsType.None, displayName : DisplayUnitsType[DisplayUnitsType.None]},
-    {value: DisplayUnitsType.Thousands, displayName : DisplayUnitsType[DisplayUnitsType.Thousands]},
-    {value: DisplayUnitsType.Millions, displayName : DisplayUnitsType[DisplayUnitsType.Millions]},
-    {value: DisplayUnitsType.Billions, displayName : DisplayUnitsType[DisplayUnitsType.Billions]},
-    {value: DisplayUnitsType.Triilions, displayName : DisplayUnitsType[DisplayUnitsType.Triilions]}
+    {value: DisplayUnitsType.Auto, displayName : "Visual_DisplayUnits_Auto"},
+    {value: DisplayUnitsType.None, displayName : "Visual_DisplayUnits_None"},
+    {value: DisplayUnitsType.Thousands, displayName : "Visual_DisplayUnits_Thousands"},
+    {value: DisplayUnitsType.Millions, displayName : "Visual_DisplayUnits_Millions"},
+    {value: DisplayUnitsType.Billions, displayName : "Visual_DisplayUnits_Billions"},
+    {value: DisplayUnitsType.Trillions, displayName : "Visual_DisplayUnits_Trillions"}
 ]
 
 const positionOptions : IEnumMember[] = [
```

Comment:
why do we have IEnumMember type here? It is clear that the values have displayNameKey and not actual displayName here. Do we really need to follow IEnumMember interface?

Reply:
because items of ItemDropDown settings is IEnumMember[] type, and it has only displayName property, that is why we need to follow IEnumMember interface